### BIG differences, this is diff of wrong files.
### CLUS "System Class LOGICAL-PATHNAME"
### and CLHS topic selection, http://clhs.lisp.se/Body/a_logica.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:logical-pathname.txt	2017-04-27 21:46:48.000000000 +0100
+++ saved-pages-clus/cl:types:logical-pathname.txt	2017-04-27 21:46:48.000000000 +0100
@@ -1,4 +1,27 @@
-LOGICAL-PATHNAME
-Please select which reference to LOGICAL-PATHNAME you intended
+System Class LOGICAL-PATHNAME

-FunctionSystem Class
\ No newline at end of file
+Class Precedence List
+logical-pathname, pathname, t
+Description
+A pathname that uses a namestring syntax that is implementation-independent, and that has component values that are implementation-independent. Logical pathnames do not refer directly to filenames
+See Also
+ {\secref\FileSystemConcepts}
+ {\secref\SharpsignP}
+ {\secref\PrintingPathnames}
+\
\ No newline at end of file
